src/cosl/rules.py

```diff
@@ -110,14 +110,24 @@
     },
     host_metrics_missing={
         "alert": "HostMetricsMissing",
-        # We use "absent(up)" with "for: 5m" because the alert transitions from "Pending" to "Firing".
-        # If query portability is desired, "absent_over_time(up[5m])" is an alternative.
         "expr": "absent(up)",
         "for": "5m",
-        "labels": {"severity": "critical"},
+        "labels": {
+            "severity": "critical"
+        },  # The remote writer will set this to critical for machine charms when initializing PrometheusRemoteWriteConsumer.
```

Comment:
Is this comment relevant here? The severity is already critical.
Iirc we said we'd replicate HostMetricsMissing (this) alert?
- Host = Unit
- Aggregator = App